docs/content/guides/author-apps/kubernetes/how-to-kubernetes-resource/index.md

```diff
@@ -0,0 +1,118 @@
+---
+type: docs
+title: "How-To: Use a Kubernetes resources in your application"
+linkTitle: "Use a Kubernetes resources in your application"
+description: "Learn how to use a Kubernetes resources in your application"
+weight: 200
+categories: "How-To"
+tags: ["Kubernetes"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Leverage Kubernetes resources in your Radius Application directly.
+
+## Prerequisites
+
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Step 1: Start with a container
```

Comment:
I think the step of adding a Radius container would be better below step 3. As written there's an implicit assumption that you have to import radius and define a container before you add Kubernetes resources. By putting it after the Kubernetes resource and instead framing the step as "Add a container and use the secret you just defined", it will be clearer that they are separate.